frontend/components/profile/ViewProfile.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { View, Text, Button, Image, StyleSheet, Dimensions } from 'react-native';
+
+export default ViewProfile = ({ navigation }) => {
+  const dWidth = Dimensions.get('window').width;
+  const dHeight = Dimensions.get('window').height;
+
+  return (
+    <View style={styles.moreInfo}>
+      <Button title="Log Out" style={{
+        float: 'right',
+        width: 50,
+        backgroundColor: '#fff',
+        color: '#000'
+      }} />
+      <Image style={{
+        alignSelf: 'center',
+        zIndex: 2,
+        width: 100,
+        height: 100,
+        borderRadius: 50,
+        margin: 10,
+        borderRadius: 20
+      }}
+      source={'https://www.akc.org/wp-content/uploads/2017/11/Longhaired-Dachshund-standing-outdoors.jpg'} />
```

Comment:
For the image to show up, source needs to be reformatted, see below: 
`source={{uri: 'https://www.akc.org/wp-content/uploads/2017/11/Longhaired-Dachshund-standing-outdoors.jpg'}}`